airbyte-cdk/java/airbyte-cdk/db-destinations/build.gradle

```diff
@@ -9,6 +9,9 @@ java {
 }
 
 dependencies {
+    compileOnly libs.lombok
```

Comment:
can't we have those only in the build.gradle of the java cdk, rather than in each cdk submodule?

Reply:
@stephane-airbyte I think that this can happen eventually when all of the CDK modules have been migrated to Micronaut AND we can also declare/add the required Micronaut annotation processors in the main CDK build.gradle.  The order that they are declared matters, so putting the lombok annotation processor in the main build.gradle, but declaring the Micronaut annotation processors in the individual module build files may not keep the order/we would be at the mercy of how Gradle merges the two and I don't believe that they make any guarantees about order.  That being said, we could add the Micronaut annotation processor to the root CDK build file now, but I was trying to not pollute classpaths yet as part of this first pass.

Reply:
let's move everything to the root CDK build file. It'll reduce the friction to adopting micronauts everywhere

Reply:
@stephane-airbyte I have done this in a later layer in this PR.